index.html

```diff
@@ -0,0 +1,186 @@
+<!DOCTYPE html>
+<html lang="pl">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>task - html-and-css</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400;500;600;700&display=swap"
+        rel="stylesheet">
+    <link rel="stylesheet" href="styles/style.css">
+</head>
+
+<body>
+
+    <header class="header" id="home">
+        <div class="wrapper">
+            <nav class="header__nav">
+                <div class="header__nav--logo">
```

Comment:
Skoro modyfikator to musi być element - to on ma główne style, a modyfikator tylko specjalne czyli `eader__nav header__nav--logo`

PS. Moim zdaniem nie pasuje tutaj modyfikator, bo nie ma "specjalnych styli". Powinien być nowy element tj. `header__logo`